controllers/topics.controller.js

```diff
@@ -40,3 +40,22 @@ exports.getArticlesInOrder = (request, response, next) => {
     });
 
 };
+
+
+exports.getCommentsById = (request, response, next) => {
+
+    const { article_id } = request.params;
+
+    selectCommentsById(article_id)
+    .then((commentArray) => {
```

Comment:
maybe change this to just 'comments', so then the array is on a key of 'comments' which will help with readability :)